test/app_web/api_test.exs

```diff
@@ -0,0 +1,66 @@
+defmodule AppWeb.APITest do
+  use AppWeb.ConnCase
+  import Mox
+
+  @create_attrs %{
+    "image" => %Plug.Upload{
+      content_type: "image/png",
+      filename: "phoenix.png",
+      path: [:code.priv_dir(:app), "static", "images", "phoenix.png"] |> Path.join()
+    }
+  }
+
+  @invalid_attrs %{
+    "image" => %Plug.Upload{
+      content_type: "application/pdf",
+      filename: "some_pdf.pdf",
+      path: [:code.priv_dir(:app), "static", "images", "some.pdf"] |> Path.join()
+    }
+  }
+
+  @invalid_field %{
+    "invalid" => %Plug.Upload{
+      content_type: "application/pdf",
+      filename: "some_pdf.pdf",
+      path: [:code.priv_dir(:app), "static", "images", "some.pdf"] |> Path.join()
+    }
+  }
+
+
+  test "upload fails", %{conn: conn} do
+    # Stub the ExAws behaviour
+    stub(ExAws.Request.HttpMock, :request, fn
+      _method, _url, _body, _headers, _opts ->
+           {:ok, %{status_code: 400, body: "error"}}
+    end)
+
+    conn = post(conn, ~p"/api/images", @create_attrs)
```

Comment:
Why does a test with the `@create_attrs`, which appear to be _valid_ attributes, result in a `400` response? 🤷‍♂️

Reply:
Fixed. ✅